oteapi/strategies/download/file.py

```diff
@@ -31,6 +31,10 @@ class FileConfig(BaseModel):
             "Encoding used when opening the file. The default is platform dependent."
         ),
     )
+    datacache_config: Optional[DataCacheConfig] = Field(
```

Comment:
```suggestion
    datacache_config: DataCacheConfig = Field(
```
As you're providing a default value, which is a default initialization of `DataCacheConfig`, there is no need to allow this field to be `None` by adding `Optional[]`.